### DISCUSSION TITLE
using composition api with quasar v1 , since vue 2 has plan for adding composition api to it too.

### DISCUSSION THREAD

**Discussion by AlaaDq** [quasarframework/quasar#12526] [Ideas / Proposals]
using composition api with quasar v1 , since vue 2 has plan for adding composition api to it too.

**Comment by MilosPaunovic** [comment]
The end of `LTS` for `Q1` is announced to be a month and a half from now (2022-04-01), so I wouldn't count on that happening.
[https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule](https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule)
